include/nigiri/rt/rt_timetable.h

```diff
@@ -35,11 +38,15 @@ using change_callback_t =
 //   up with their trip_id in the RT timetable.
 struct rt_timetable {
   rt_transport_idx_t add_rt_transport(
-      source_idx_t const,
+      source_idx_t,
       timetable const&,
-      transport const,
-      std::span<stop::value_type> const& stop_seq = {},
-      std::span<delta_t> const& time_seq = {});
+      transport,
+      std::span<stop::value_type> const stop_seq = {},
+      std::span<delta_t> const time_seq = {},
+      std::string_view const new_trip_id = {},
+      std::string_view const route_id = {},
+      std::string_view const display_name = {},
```

Comment:
```suggestion
      std::span<stop::value_type> stop_seq = {},
      std::span<delta_t> time_seq = {},
      std::string_view new_trip_id = {},
      std::string_view route_id = {},
      std::string_view display_name = {},
```

`span` and `string_view` are const anyway (no non-const acessors)